### PR TITLE
Setup my  profile for React Dev Squad

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
Reviewers, please check my PR

### CODE REVIEW

File: ids/mapping.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "nikhil": "whydonti",
   "om": "strider99",
   "prakash": "prakashchoudhary07",
+  "Prem": "PremanandChowdhury",
```
Comment (Rucha1499) [inline ids/mapping.json:20]: Can you change it to ` prem `, following the conventions?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thank you for this pull request! 😄 

Requesting changes in the `id` field
Also, please rename the folder name correspondingly

One more thing about the profile photo is that:
>The face should be visible, centered and identifiable
Face should cover at least 60% of the image

Rules for RDS username and profile photo: https://github.com/Real-Dev-Squad/website-static#rules-for-member_rds_id-in-real-dev-squad

File: members/Prem/data.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "id": "Prem",
```
Comment (Rucha1499) [inline members/Prem/data.json:2]: same change here! 🙂

Comment (PremanandChowdhury) [reply members/Prem/data.json:2]: ok

[the remark this answers is not in the document]
Comment (PremanandChowdhury) [reply members/Prem/data.json:2]: ok

**Pull request closed without merging**